docs/source/conf.py

```diff
@@ -103,6 +103,9 @@
 html_static_path = ["_static"]
 html_css_files = ["css/custom.css"]
 
+# Remove "Edit on GitHub"
```

Comment:
Could you add a period to follow the coding-style conventions?
https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#block-and-inline-comments-are-supposed-to-be-sentences